experiment_handling.py

```diff
@@ -121,10 +125,18 @@ def resave_data(save_path, parameter_combinations, index, eva, name,
         key that misses in the result dictionaries for badruns (Default: None)
     """
     # add "/" to save_path if it does not end already with a "/"
-    save_path += "/" if not save_path.endswith("/") else ""
+    source_path += "/" if not source_path.endswith("/") else ""
+
+    # default save_path to source_path if not save_path is given
+    if save_path == None:
```

Comment:
These tow lines could be avoided by giving the default data directory in the function definition above, if I'm not completely wrong now